site/content/_future.txt

```diff
@@ -12,6 +12,7 @@
       <a href="/events/2015-toronto/">Toronto - May 2015</a><br/>
       <a href="/events/2015-derby/">Derby - Spring 2015</a><br/>
       <a href="/events/2015-washington-dc/">Washington, DC - Spring 2015</a><br/>
+      <a href="/events/2015-singapore/">Singapore 2015</a><br/>
```

Comment:
You have a merge conflict in this file. Please ensure you're starting from the most current version in https://github.com/jedi4ever/devopsdays-webby/blob/master/site/content/_future.txt for your changes; please fix the merge conflict in your PR. Thanks.